src/main/java/com/gorlah/kappabot/rest/validation/ValidationErrorResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.gorlah.kappabot.rest.validation;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Value;
+
+import java.util.List;
+
+@Value
+class ValidationErrorResponse {
```

Comment:
That would actually be pretty cool. Do you basically just write a `ControllerAdvice` that would handle `Exception.class`? Then return the exception's message as a 500? Although I guess some exceptions could be more suitable for 400 exceptions.

Reply:
Yeah, you would have some kind of advice or other exception handling mechanism that handles all `Exception` and you can have more specific things like your validation errors still be handled specially, and then in general fall back to returning a 500 error and maybe a concise message, but if you have all your errors return a consistent model it can make things easier for your API clients. 

One thing I recently started including in my error objects that I am happy with is a unique reference ID that is generated when the exception is being translated into my error object. That way I can leave the stacktrace out of the response but still be able to tie that response back to the stacktrace by searching the server log for the unique ID.